tests/app/authentication/test_authentication.py

```diff
@@ -309,7 +311,7 @@ def test_should_attach_the_current_api_key_to_current_app(notify_api, sample_ser
             headers={'Authorization': 'Bearer {}'.format(token)}
         )
         assert response.status_code == 200
-        assert api_user == sample_api_key
+        assert api_user.id == sample_api_key.id
```

Comment:
You switched to using `AuthenticatedServiceApiKey` for `api_user` because of the way you build `AuthenticatedServiceInfo`. These tests should not have changed. What should have changed is that we shouldn't use `sample_api_key`, we should have a new fixture for this use case.